application-admintools-ui/src/main/resources/AdminTools/Code/Configuration.xml

```diff
@@ -334,7 +334,22 @@
       <categoryIcon/>
     </property>
     <property>
-      <codeToExecute/>
+      <codeToExecute>#set ($adminToolsConfigClassName = 'AdminTools.Code.ConfigurationClass')
+#set ($classExists = $xwiki.exists($adminToolsConfigClassName))
+#if (!$classExists)
```

Comment:
I know it's minor, but you could simply write `#if (!$xwiki.exists($adminToolsConfigClassName))`, since `$classExists` is not used anywhere else, and it's also not long